extensions/2.0/Vendor/FB_geometry_metadata/README.md

```diff
@@ -0,0 +1,108 @@
+# FB\_geometry\_metadata
+
+## Contributors
+
+* Pär Winzell, Facebook, [@zellski](https://twitter.com/zellski)
+* Andrew Prasetyo Jo, Facebook
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension annotates glTF `scene` objects with a summary of the cumulative geometric complexity and world-space extents of the scene's associated scene graph. Each recursively referenced mesh is enumerated for total vertex and primitive count, and each individual vertex is transformed into world-space for a perfectly computed bounding box.
+
+Some proven real-world applications for this information:
+- When a viewer needs to make a quick, high-level decision which `scene` to display. This is perhaps most meaningful e.g. when `scenes` are used to represent different variants (e.g. LoD levels) of a model.
+- When server-side systems need to make decisions based on geometric complexity, and don't want to have to include their own complex matrix/vector code.
+
+## Specifying Geometry Data
+
+The summary data is defined as an optional extension to a glTF `scene`, by adding an `extensions` property and defining a `KHR_lights_punctual` property with a `lights` array inside it:
+
+```javascript
+"extensions": {
+    "FB_geometry_metadata" : {
+        "vertexCount": 28727,
+        "primitiveCount": 15451,
+        "worldBounds": {
+            "min": [
+                -3.0651053919852,
+                0,
+                -2.6551087079312
+            ],
+            "max": [
+                3.0651053919852,
+                9.8512265772065,
+                2.6538096091834
+            ]
+        }
+    }
+}
+```
+
+## Limitations
+This extension completely disregards animation: the static transforms, as supplied on each `node`, are taken at face value. No skeletal animations are applied, nor any morph targets.
```

Comment:
I assume this means that no animation is applied when _reading_ the bounds? Or would you also like to impose that animation should not be applied when writing the bounds? Either way, may be good to specify here.

Reply:
Hmm. I meant that these values were computed without applying any animation-related effects, and that a reader should interpret them accordingly. You're right, should be more specific.